tests/javascript_tests/test_color_view_controller.js

```diff
@@ -126,7 +126,7 @@ requirejs([
 
     test("Test ColorViewController.getColorList exceptions", function(){
       var five;
-      five = [0, 1, 2, 3, 4];
+      five = ['0', 'string1', 'string2', 'string3', 'string4'];
```

Comment:
I see. I think that suggestion is good, could probably mean just
re-purposing some of the already existing tests, and making sure all
paths are covered.

On (May-09-16| 8:50), Joshua Shorenstein wrote:
> > @@ -126,7 +126,7 @@ requirejs([
> >  
> >      test("Test ColorViewController.getColorList exceptions", function(){
> >        var five;
> > -      five = [0, 1, 2, 3, 4];
> > +      five = ['0', 'string1', 'string2', 'string3', 'string4'];
> 
> No actual functionality was changed save that error throwing, so the tests made sure it still works the same. I can add helper function tests though.
> 
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/468/files/c5c2d10589eb2a5c7dd4d326dfa32745e0ee82cc#r62523101